composer.json

```diff
@@ -1,159 +1,116 @@
 {
-  "name": "skilld-labs/sdc",
-  "description": "Project template for Drupal 8 projects with composer",
-  "type": "project",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "",
-      "role": ""
-    }
-  ],
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.drupal.org/8"
-    },
-    {
-      "type": "package",
-      "package": {
-        "name": "ckeditor/panelbutton",
-        "version": "4.10.1",
-        "type": "drupal-library",
-        "dist": {
-          "url": "https://download.ckeditor.com/panelbutton/releases/panelbutton_4.10.1.zip",
-          "type": "zip"
+    "name": "skilld-labs/sdc",
```

Comment:
Please keep same tabbing as on previous composer.json file to easily identify the changes

Reply:
Btw the right name is https://packagist.org/packages/skilld-labs/sdc

Reply:
Not clear why indent changed from 2 to 4?!